commands/sunsubscribe.md

```diff
@@ -0,0 +1,3 @@
+Unsubscribes the client from the given shard channels, or from all of them if none is given.
+
+When no shard channels are specified, the client is unsubscribed from all the previously subscribed channels. In this case, a message for every unsubscribed channel will be sent to the client.
```

Comment:
This only unsubscribes from shard channels right? Probably worth being explicit here about that.